incuna_auth/middleware/login_required.py

```diff
@@ -57,6 +57,9 @@ def __init__(self, check=True):
         if check:
             check_request_has_user()
 
+    def get_unauthorised_redirect_url(self, request):
+        return settings.LOGIN_REDIRECT_URL
```

Comment:
How often are we likely to not use the `request` when overriding this? I'm wondering if the base implementation should look something like:
```python
class BasePermissionMiddleware:
    unauthorised_redirect_url = '/'

    def get_unauthorised_redirect_url(self, request):
        return self.unauthorised_redirect_url
```

This would allow this class to do:

```python
class LoginRequiredMiddleware(...):
    unauthorised_redirect_url = settings.LOGIN_REDIRECT_URL
```